.devcontainer/devcontainer.json

```diff
@@ -5,25 +5,30 @@
 	"dockerComposeFile": "docker-compose.yml",
 	"service": "app",
 	"workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
-
 	// Features to add to the dev container. More info: https://containers.dev/features.
 	// "features": {},
-
 	// Configure tool-specific properties.
 	"customizations": {
 		// Configure properties specific to VS Code.
 		"vscode": {
 			// Add the IDs of extensions you want installed when the container is created.
-			"extensions": ["mongodb.mongodb-vscode", "octref.vetur", "mikestead.dotenv", "dbaeumer.vscode-eslint"]
+			"extensions": [
+				"mongodb.mongodb-vscode",
+				"octref.vetur",
```

Comment:
Could remove vetur. Only needed for Vue, but that's in the other repo

Reply:
Can you add esbenp.prettier-vscode instead for formatting

Reply:
Updated the packages :)